operator/tools/operator-sdk/go.mod

```diff
@@ -1,8 +1,6 @@
 module github.com/stackrox/rox/operator/tools/operator-sdk
 
-go 1.22
-
-toolchain go1.22.5
+go 1.23.2
```

Comment:
I'd usually say we should do:

```
go 1.23

toolchain 1.23.2
```

but I won't block this PR over this, especially when it's not the main module